src/world/footpath.c

```diff
@@ -1740,9 +1740,9 @@ void footpath_update_path_wide_flags(int x, int y)
 	footpath_clear_wide(x, y);
 	y -= 0x20;
 
-	if (!(x & 0xE0))
+	if (!(x & 0xFFE0))
 		return;
-	if (!(y & 0xE0))
+	if (!(y & 0xFFE0))
```

Comment:
Line 1758 of this code I think could be improved whilst we are at it. At present a single tile with no connections is classed as wide. This does not matter for AI as a single tile will never be pathed onto but it does matter for performance reasons so we might as well just continue the loop if there are no edges:
```
        if ((mapElement->properties.path.edges & 0xF) == 0)
            continue;
```